rust/src/transformations/make_stable_expr/mod.rs

```diff
@@ -94,11 +97,56 @@ where
 {
     fn make_stable(
         self,
-        _input_domain: ExprDomain,
-        _input_metric: M,
+        input_domain: ExprDomain,
+        input_metric: M,
     ) -> Fallible<Transformation<ExprDomain, ExprDomain, M, M>> {
         match self {
+            #[cfg(feature = "contrib")]
+            Expr::Column(_) => expr_col::make_expr_col(input_domain, input_metric, self),
             _ => return fallible!(MakeTransformation, "Unsupported expr"),
         }
     }
 }
+
+#[cfg(test)]
+pub mod polars_test {
+
+    use std::collections::BTreeSet;
+
+    use crate::domains::{
+        AtomDomain, ExprContext, ExprDomain, Frame, FrameDomain, LazyFrameDomain, Margin,
+        SeriesDomain,
+    };
+    use crate::error::*;
+    use polars::prelude::*;
+
+    pub fn get_test_data() -> Fallible<(LazyFrameDomain, LazyFrame)> {
+        let lf_domain = LazyFrameDomain::new(vec![
+            SeriesDomain::new("A", AtomDomain::<i32>::default()),
+            SeriesDomain::new("B", AtomDomain::<f64>::new_closed((0.5, 2.5))?),
+            SeriesDomain::new("C", AtomDomain::<i32>::default()),
+        ])?
+        .with_margin::<&str>(&[], Margin::new().with_max_partition_length(3u32))?
+        .with_margin(&["A"], Margin::new().with_max_partition_length(2u32))?
+        .with_margin(&["B"], Margin::new().with_max_partition_length(2u32))?
+        .with_margin(&["C"], Margin::new().with_max_partition_length(1u32))?;
```

Comment:
I don't think "C" is referenced elsewhere -- If it's doing something interesting, add tests, if it's not, then remove?

"A" and "B" are both referenced: assuming the differences between then are significant, maybe have the names reflect that, so if a test fails for one and not the other, you have an clue about how they are different, without reading this code, or going into the debugger?

Something like:
`A` -> `open_partition_2`
`B` -> `closed_partition_2`
`C` -> `open_partition_1`